providers/code-signing-services-free.md

```diff
@@ -22,22 +22,21 @@ AltStore a longside AltServer, is a custom self-hosted app store which runs on y
 
 
 
-![appdb Rickpactor](../.gitbook/assets/rickpactor_80px.png)
+![appdb](../.gitbook/assets/appdb\_logo-svg\_56px.png)
 
-## [appdb Rickpactor](https://appdb.to/app/standalone/1900000033) or [Free signing](https://appdb.to/news/463)
+## [appdb](https://appdb.to/news/463)
 
-appdb Rickpactor is a tool to install apps from appdb and your own apps from appdb's MyAppStore for free. Similar to Cydia Impactor, but actively maintained. Now supporting Apple's Silicon M1. **macOS/PC required** for initial app installation**.**
+appdb offers free signing for everyone with **community-provided certificate and provisioning profile**. As appdb uses official Apple protocols to deploy apps to devices, there is no need to manually trust it and everything works out of the box.
```

Comment:
`Appdb offers free signing using **[enterprise certificates](https://sideloading.gitbook.io/sideloading-master-guide/providers/code-signing-services-free#enterprise-certificates) and a provisioning profile**. As Appdb uses Apple's Developer API to install apps to devices, there is no need to manually trust the apps. `